client/FxAccountClient.js

```diff
@@ -905,6 +905,123 @@ define([
       });
   };
 
+  /**
+   * @method sessionReauth
+   * @param {String} sessionToken sessionToken obtained from signIn
+   * @param {String} email Email input
+   * @param {String} password Password input
+   * @param {Object} [options={}] Options
+   *   @param {Boolean} [options.keys]
+   *   If `true`, calls the API with `?keys=true` to get the keyFetchToken
+   *   @param {Boolean} [options.skipCaseError]
+   *   If `true`, the request will skip the incorrect case error
+   *   @param {String} [options.service]
+   *   Service being accessed that needs reauthentication
+   *   @param {String} [options.reason]
+   *   Reason for reauthentication. Can be one of: `signin`, `password_check`,
+   *   `password_change`, `password_reset`
+   *   @param {String} [options.redirectTo]
+   *   a URL that the client should be redirected to after handling the request
+   *   @param {String} [options.resume]
+   *   Opaque url-encoded string that will be included in the verification link
+   *   as a querystring parameter, useful for continuing an OAuth flow for
+   *   example.
+   *   @param {Object} [options.metricsContext={}] Metrics context metadata
+   *     @param {String} options.metricsContext.deviceId identifier for the current device
+   *     @param {String} options.metricsContext.flowId identifier for the current event flow
+   *     @param {Number} options.metricsContext.flowBeginTime flow.begin event time
+   *     @param {Number} options.metricsContext.utmCampaign marketing campaign identifier
+   *     @param {Number} options.metricsContext.utmContent content identifier
+   *     @param {Number} options.metricsContext.utmMedium acquisition medium
+   *     @param {Number} options.metricsContext.utmSource traffic source
+   *     @param {Number} options.metricsContext.utmTerm search terms
+   *   @param {String} [options.unblockCode]
```

Comment:
Looks like `.signIn` and this didn't include the `verificationMethod` and `originalLoginEmail` options.